### PR TITLE
Compare Labels in different incidents

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/16654

## Description
New script that takes 2 incidents and compares their labels.

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

**Comment by orlichter1** [comment]
@kirbles19 
New script, please review in script yml. 
Description (script and arguments)

**Comment by kirbles19** [comment]
@orlichter1 Please provide links to the files I need to review/edit.

**Comment by orlichter1** [comment]
@kirbles19 you're right sorry. Here is the link:
https://github.com/demisto/content/blob/Comparelabels/Scripts/CompareIncidentsLables/CompareIncidentsLabels.yml

**Comment by kirbles19** [comment]
@orlichter1 All done reviewing the .yml.

File: Scripts/CompareIncidentsLables/CompareIncidentsLabels.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+try:
+    inc1 = demisto.args().get('inc1')
+    inc2 = demisto.args().get('inc2')
+    inc1_data = demisto.executeCommand("getIncidents", {'id': inc1})[0].get('Contents').get('data')
```
Comment (yaakovi) [inline Scripts/CompareIncidentsLables/CompareIncidentsLabels.py]: before try to access the result, check to see if the command succeeded.
use:
```
res = demisto.executeCommand("getIncidents", {'id': inc1})
if any(is_error(entry) for entry in res):
    <insert error handling here>
inc1_data = res[0].get('Contents').get('data')
```

File: Scripts/CompareIncidentsLables/CompareIncidentsLabels.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    inc2_data = demisto.executeCommand("getIncidents", {'id': inc2})[0].get('Contents').get('data')
+    if inc1_data is None or inc2_data is None:
+        return_error("One of the incidents does not exist.")
+
+    inc1_labels = inc1_data[0].get('labels')
```
[the remark this answers is not in the document]
Comment (orlichter1) [reply Scripts/CompareIncidentsLables/CompareIncidentsLabels.py]: Changed. But whats the difference?

**Pull request merged**